app/javascript/.eslintrc.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
               '@utilities': path.join(__dirname, './utilities'),
               '@components': path.join(__dirname, './shared/components'),
               '@images': path.join(__dirname, '../assets/images'),
-              '@admin-controllers': path.join(__dirname, './admin/controllers'),
```

Comment:
I removed this alias, because we now have some components, not just Stimulus controllers.

Reply:
#16412 covers this.